README.md

```diff
@@ -121,17 +121,113 @@ Resolved environments can also be created via the API:
 
 ## Quickstart
 
-First, install Rez. Download the source, and from the source directory, run
-(with DEST_DIR replaced with your install location):
+Choose your installation method.
 
-    ]$ python ./install.py -v DEST_DIR
+<details>
+  <summary>Simple</summary>
+<table>
+<tr><td>
+<br>
+
+**Simple** The simple approach is well suited for beginners and those looking to learn more about Rez and whether it is suited to their problem and environment.
+
+</td></tr>
+<tr><td>
+
+```bash
+$ pip install rez --user
+```
+</td></tr>
+<tr><td>
+
+<br>
+
+**Advantages**
+
+- **User-friendly and familiar installation method** Everybody loves pip
+
+</td></tr>
+</table>
+
+</details>
+</details><details>
+  <summary>Recommended</summary>
+
+<table>
+<tr><td>
+<br>
+
+**Recommended** The recommended approach is better suited for live production, where you've committed to Rez and want precision and control.
+
+</td></tr>
+<tr><td>
+
+```bash
+$ python -m virtualenv rez
+$ rez\Scripts\activate
+(rez) $ pip install rez
+```
+
+</td></tr>
+<tr><td>
+<br>
+
+**Optional**
+
+In case you also want to make `python` into a Rez package, you may also want to remove any reference to `python` from your virtual environment.
+
+```bash
+# Windows
+(rez) $ del rez\Scripts\python.exe
+
+# Linux
+(rez) $ rm rez\bin\python
```

Comment:
This won't work. And actually, it would break the entire thing. The scripts installed by your pip package directly link to this specific python.

Like here is the `pylint` binary created when installing pylint inside a virtualenv:

```python
#!/mnt/users/jcmorin/.local/share/virtualenvs/tractor-to-elasticsearch-T8zSJag0/bin/python2

# -*- coding: utf-8 -*-
import re
import sys

from pylint import run_pylint

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(run_pylint())
```
```
lrwxrwxrwx 1 jcmorin artists     7 Apr 25 17:29 python -> python2
-rwxrwxr-x 1 jcmorin artists  7216 Apr 25 17:29 python2
lrwxrwxrwx 1 jcmorin artists     7 Apr 25 17:29 python2.7 -> python2
```
Well, it will "work" but it's a really really dirty hack. Because the recommended way for launching python is to use `python2` or `python3` (so we are sure which version is really used).

So it doesn't cover all cases.